File: fuzzer/compiled_programs/gen_apLDkXfYEu.py
from sklearn import *
import pandas as pd

YSkeZKjE = pd.read_csv("../datasets/random-dataset__Class.csv")
YSkeZKjE = YSkeZKjE.dropna()

YSkeZKjE_Y = YSkeZKjE["Class"]
YSkeZKjE = YSkeZKjE.drop(columns=["Class"])

YSkeZKjE_scaler = preprocessing.MinMaxScaler()
YSkeZKjE = YSkeZKjE_scaler.fit_transform(YSkeZKjE)

LpVhOdi = pd.read_csv("../datasets/Iris__Species.csv")
LpVhOdi = LpVhOdi.dropna()

LpVhOdi_Y = LpVhOdi["Species"]
LpVhOdi = LpVhOdi.drop(columns=["Species"])

HACWt = neural_network.MLPClassifier()

saVRw = neighbors.KNeighborsClassifier()

YSkeZKjE_X_train, YSkeZKjE_X_test, YSkeZKjE_Y_train, YSkeZKjE_Y_test = model_selection.train_test_split(YSkeZKjE, YSkeZKjE_Y, test_size = 0.7)
saVRw.fit(YSkeZKjE_X_train, YSkeZKjE_Y_train)

print("Accuracy score : " + str(metrics.accuracy_score(YSkeZKjE_Y_test, saVRw.predict(YSkeZKjE_X_test))))

YSkeZKjE_X_train, YSkeZKjE_X_test, YSkeZKjE_Y_train, YSkeZKjE_Y_test = model_selection.train_test_split(YSkeZKjE, YSkeZKjE_Y, test_size = 0.7)
saVRw.fit(YSkeZKjE_X_train, YSkeZKjE_Y_train)

print("Accuracy score : " + str(metrics.accuracy_score(YSkeZKjE_Y_test, saVRw.predict(YSkeZKjE_X_test))))

LpVhOdi_X_train, LpVhOdi_X_test, LpVhOdi_Y_train, LpVhOdi_Y_test = model_selection.train_test_split(LpVhOdi, LpVhOdi_Y, test_size = 0.7)
saVRw.fit(LpVhOdi_X_train, LpVhOdi_Y_train)

print("Accuracy score : " + str(metrics.accuracy_score(LpVhOdi_Y_test, saVRw.predict(LpVhOdi_X_test))))

LpVhOdi_X_train, LpVhOdi_X_test, LpVhOdi_Y_train, LpVhOdi_Y_test = model_selection.train_test_split(LpVhOdi, LpVhOdi_Y, test_size = 0.7)
HACWt.fit(LpVhOdi_X_train, LpVhOdi_Y_train)

print("Accuracy score : " + str(metrics.accuracy_score(LpVhOdi_Y_test, HACWt.predict(LpVhOdi_X_test))))

File: fuzzer/compiled_programs/gen_apLDkXfYEu.r
library(tidyr)
suppressMessages(library(dplyr))
library(e1071) # SVM implementation
library(class) # KNN implementation
library(rpart) # DecisionTree implementation
library(nnet) # MLP implementation
suppressMessages(library(caret))

YSkeZKjE <- read.csv("../datasets/random-dataset__Class.csv")
YSkeZKjE <- YSkeZKjE %>% drop_na()

YSkeZKjE_X <- subset(YSkeZKjE, select = -Class)
YSkeZKjE_Y <- as.factor(YSkeZKjE$Class)

preprocess_range <- preProcess(YSkeZKjE_X, method = "range")
YSkeZKjE_X <- predict(preprocess_range, newdata = YSkeZKjE_X)

LpVhOdi <- read.csv("../datasets/Iris__Species.csv")
LpVhOdi <- LpVhOdi %>% drop_na()

LpVhOdi_X <- subset(LpVhOdi, select = -Species)
LpVhOdi_Y <- as.factor(LpVhOdi$Species)

train_index <- sample(1:nrow(YSkeZKjE), (1-0.7) * nrow(YSkeZKjE))
YSkeZKjE_X_train <- YSkeZKjE_X[train_index, ]
YSkeZKjE_X_test <- YSkeZKjE_X[-train_index, ]
YSkeZKjE_Y_train <- YSkeZKjE_Y[train_index]
YSkeZKjE_Y_test <- YSkeZKjE_Y[-train_index]

saVRw <- knn(train = YSkeZKjE_X_train, test = YSkeZKjE_X_test, cl = YSkeZKjE_Y_train)

train_index <- sample(1:nrow(YSkeZKjE), (1-0.7) * nrow(YSkeZKjE))
YSkeZKjE_X_train <- YSkeZKjE_X[train_index, ]
YSkeZKjE_X_test <- YSkeZKjE_X[-train_index, ]
YSkeZKjE_Y_train <- YSkeZKjE_Y[train_index]
YSkeZKjE_Y_test <- YSkeZKjE_Y[-train_index]

saVRw <- knn(train = YSkeZKjE_X_train, test = YSkeZKjE_X_test, cl = YSkeZKjE_Y_train)

train_index <- sample(1:nrow(LpVhOdi), (1-0.7) * nrow(LpVhOdi))
LpVhOdi_X_train <- LpVhOdi_X[train_index, ]
LpVhOdi_X_test <- LpVhOdi_X[-train_index, ]
LpVhOdi_Y_train <- LpVhOdi_Y[train_index]
LpVhOdi_Y_test <- LpVhOdi_Y[-train_index]

saVRw <- knn(train = LpVhOdi_X_train, test = LpVhOdi_X_test, cl = LpVhOdi_Y_train)

train_index <- sample(1:nrow(LpVhOdi), (1-0.7) * nrow(LpVhOdi))
LpVhOdi_X_train <- LpVhOdi_X[train_index, ]
LpVhOdi_X_test <- LpVhOdi_X[-train_index, ]
LpVhOdi_Y_train <- LpVhOdi_Y[train_index]
LpVhOdi_Y_test <- LpVhOdi_Y[-train_index]

HACWt <- nnet(LpVhOdi_Y_train ~ ., LpVhOdi_X_train, size = 8)

